tensorflow/python/ops/script_ops_test.py

```diff
@@ -37,6 +39,36 @@ def plus(a, b):
     expect_result = constant_op.constant(3, dtypes.int32)
     self.assertAllEqual(actual_result, expect_result)
 
+  def test_stateless_flag(self):
+
+    def plus(a, b):
+      return a + b
+
+    @def_function.function
+    def tensor_plus_stateful(a, b):
+      return numpy_function(plus, [a, b], dtypes.int32, stateful=True)
+
+    @def_function.function
+    def tensor_plus_stateless(a, b):
+      return numpy_function(plus, [a, b], dtypes.int32, stateful=False)
+
+    @def_function.function(autograph=False)
+    def tensor_double_plus(a, b, c, d):
+      sum_stateful = tensor_plus_stateful(a, b)
+      assert sum_stateful.op.op_def.is_stateful
```

Comment:
I don't think this is a robust operation. If you prefer, set up a test with a numpy function that's called twice, and check that the python function only tuns once - that's what the contract promises.

Reply:
Indeed, I changed that to two calls